src/renderer/components/common/tooltip.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { PureComponent } from 'react'
+
+class Tooltip extends PureComponent {
+  render() {
+    const { children, tip, name } = this.props
+    return (
+      <div class="tooltip">
+        <span>{children}</span>
```

Comment:
I think is better to just remove this `span` wrapper, unless is necessary some how?